sympy/utilities/tests/diagnose_imports.py

```diff
@@ -0,0 +1 @@
+../../../bin/diagnose_imports.py
```

Comment:
Nothing of this is in the test suite.
I think this link was accidentally committed.